### ISSUE TITLE
Wiki Suggestions

### ISSUE THREAD

**Issue by Myoldmopar** [NREL/EnergyPlus#6416]
Issue overview
--------------
From a user on the Helpdesk:

- There should be more detail in describing what “source tree” refers to. The instructions, best I can tell, assume source tree means “my clone of the repository.” That’s confusing because the top level of the cloned repository has lots of subdirectories, one of which is “src”, which sure sounds like it would be the source tree. It’s a bit surprising to find things like “doc”, “design”, “weather”, etc., under something called a source tree.
- Bullet #3 under Linux could use an example. I.e., give an example location of the cloned tree and a proposed name for the build tree. That’s really part of the previous suggestion I guess.

**Comment by jasondegraw** [comment]
I think we need to be careful here, some amount of familiarity with CMake , etc. needs to be assumed. I'm all for clear build instructions, but we shouldn't produce a CMake tutorial.

**Comment by Myoldmopar** [comment]
Agreed.  I haven't actually gone back to look at the what all the wiki shows, just saving this as a placeholder.  Putting a tiny example up there with actual directory names that we label the source and build trees is within reason.
